### PR TITLE
APB-5081 trusts 5MLD URN

### PR DESCRIPTION
**Pull request by geoffreywatson**

### CODE REVIEW

**Review by Leeroyc720 — COMMENTED** [review]
Just a few commented-out code that we will need to clean-up, But LGTM👍

File: app/uk/gov/hmrc/agentinvitationsfrontend/journeys/AgentInvitationJourneyModel.scala [hunk 1]
```diff
@@ -549,6 +559,7 @@ object AgentInvitationJourneyModel extends JourneyModel with Logging {
     Transition {
       case ReviewAuthorisationsTrust(_, basket) =>
         if (confirmation.choice) {
+          println(s"basket on review authorisation is $basket")
```
Comment (Leeroyc720) [inline app/uk/gov/hmrc/agentinvitationsfrontend/journeys/AgentInvitationJourneyModel.scala:562]: just out of curiosity are we trying to review what's in the basket here?

**Review by Leeroyc720 — APPROVED** [review]

**Review by oarshad — APPROVED** [review]

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationFastTrackJourneyController.scala [hunk 2]
```diff
@@ -708,11 +711,11 @@ object AgentInvitationFastTrackJourneyController {
...
-  def IdentifyTrustClientForm: Form[TrustClient] = Form(
-    mapping(
-      "utr" -> normalizedText.verifying(validUtr())
-    )(x => TrustClient.apply(Utr(x)))(x => Some(x.utr.value))
-  )
+//  def IdentifyTrustClientForm: Form[TrustClient] = Form(
+//    mapping(
+//      "utr" -> normalizedText.verifying(validUtr())
+//    )(x => TrustClientTaxable.apply(Utr(x)))(x => Some(x.taxId.value))
+//  )
```
[the remark this answers is not in the document]
Comment (geoffreywatson) [reply app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationFastTrackJourneyController.scala]: done thx

File: app/uk/gov/hmrc/agentinvitationsfrontend/journeys/AgentInvitationJourneyModel.scala (hunk 1 above)
Comment (geoffreywatson) [reply app/uk/gov/hmrc/agentinvitationsfrontend/journeys/AgentInvitationJourneyModel.scala]: yeah that shouldn't be there have removed

**Review by Leeroyc720 — APPROVED** [review]

**Pull request merged**